SPECS/kernel/config

```diff
@@ -6292,7 +6291,6 @@ CONFIG_DELL_WMI_SYSMAN=m
 # CONFIG_X86_PLATFORM_DRIVERS_HP is not set
 # CONFIG_WIRELESS_HOTKEY is not set
 # CONFIG_IBM_RTL is not set
-# CONFIG_LENOVO_YMC is not set
```

Comment:
Result of https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/f8761b11f1c054c510d2e40784f3ee5ef94c12bf#diff-fe2c79476598550f03b23b905fe081d5a7f27387bf45a2a0b8a7e38f85769092